api/src/test/java/org/openmrs/module/mirebalaisreports/definitions/AppointmentsReportManagerTest.java

```diff
@@ -49,16 +48,17 @@
     @Test
     public void testAppointmentsExport() throws Exception {
 
+        Location mirebalais = locationService.getLocation("Mirebalais");
```

Comment:
I'm OK with these looking up by name I guess, but I'd much rather see us add an approach (as I indicate above) that allows us to reference locations in our code, and then use that approach in our unit tests, as that will both give us better coverage of testing that particular approach, and be more maintainable in the tests in case location names are changed.